rclrs/src/publisher.rs

```diff
@@ -50,9 +50,9 @@ impl Drop for PublisherHandle {
 /// The underlying RMW will decide on the concrete delivery mechanism (network stack, shared
 /// memory, or intraprocess).
 ///
-/// Sending messages does not require calling [`spin`][1] on the publisher's node.
+/// Sending messages does not require the node's executor to [spin][2].
 ///
-/// [1]: crate::spin
+/// [2]: crate::Executor::spin
```

Comment:
Nit-pick, but this could just be `[1]` instead of `[2]`.

Reply:
Done: [e7f8bac](https://github.com/ros2-rust/ros2_rust/pull/428/commits/e7f8bac0b69cdefed15267473526c69fa161f125)